### DISCUSSION TITLE
Guide to Stellar Network

### DISCUSSION THREAD

**Discussion by wildework** [stellar/sorobanathon#111] [General]
The original guides for local and Futurenet usage are great! I did add extra details and sections on the various services that compose the docker container, as well as extra information on how accounts work.

You can find the guide here https://www.sorobandev.com/guides/stellar-network

I'm also tweaking the UI to be more friendly for copy-paste heavy content, like the one in this guide. There's now a button above each code section that can use it.

**Comment by achallagundla** [comment]
Heya @L-Zuluaga ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1000 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
